thunder/tests/opinfos.py

```diff
@@ -6153,6 +6153,63 @@ def tensor_1d_sample_generator(op, device, dtype, requires_grad, **kwargs):
 nn_ops = []
 
 
+def baddbmm_sample_generator(op, device, dtype, requires_grad, **kwargs):
+    make = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+    cases = (
+        ((0, 0, 0), (0, 0, 0), (0, 0, 0)),
+        ((3, 0, 5), (3, 0, 0), (3, 0, 5)),
+        ((0, 5, 6), (0, 5, 0), (0, 0, 6)),
+        ((3, 5, 6), (3, 5, 0), (3, 0, 6)),
```

Comment:
Let's add cases where the matmul part actually produces something other than zero